dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -54,11 +53,11 @@ def self.find_by_studio_person_id(studio_person_id)
   end
 
   # @return {boolean} true if user is an eligible facilitator or attended relevant workshop
```

Comment:
Please update this doc comment to explain the new workshop_subjects parameter.

Reply:
Updated -- I'm not familiar with YARD yet (which is looks like is what we use for our docs?), so hopefully this is syntactically ok.

Reply:
:+1: Seems fine.  We're not actually generating these docs, so syntax errors in YARD are not a big deal right now.